### DISCUSSION TITLE
Scheduling content

### DISCUSSION THREAD

**Discussion by ajulvestad** [reaby/infoscreen3#21] [Ideas]
Add a feature to schedule when a slide is part of a bundle and/or when a playlist is active on a screen.

Why: At our party, we usually (try) to plan things in advance. The canteen people want to plan for a happy hour on some items, and want a slide to appear and be gone at a certain time. Altso, we've usually planned so that when it's midnight the day before closing, slides regarding "remember to clean up your stuff" appears, and at 08:00, a new slide appears: "thanks for this year!"

This would be a really nice feature.

**Comment by reaby** [comment]
This was actually requested previously as well, but i've not yet to find a good solution for it. Have to think how to make this, so it fits good on the ui...

Maybe it would make sense to introduce some sort of slide sub group/bundle/folder structure, which you could schedule with a start and end time.
Or start time and duration, with duration -1 then it gets toggled on always.

Have to think.

**Comment by reaby** [comment]
Okay, I implemented the feature.
On each slide you have start and end date/time.
